[cmake] simplify copies of misc files into build directory (#9834)
root-project/root: CMakeLists.txt

                   COMMAND ${CMAKE_COMMAND} -E touch ${stamp_file}
                   COMMENT "Copying directories such as etc, icons, fonts, js, ui5, etc. to build area")

> #---- List of directories which completely copied to target directory by each build

Reviewer: ```suggestion
#---- List of source directories which are copied verbatim into the build directory
```

> set(artifact_dirs "etc;test;icons;fonts;macros;tutorials")

Reviewer: You can probably also move the `README` directory from above (line 282) into this list.

Author: Done
